AGILE/AgileForm.cs

```diff
@@ -150,6 +151,21 @@ private void AgileForm_Load(object sender, EventArgs e)
 
             #endregion Load Screen Metrics
 
+            // Get last fullscreen state
+            if (Properties.Settings.Default.fullScreen == true)
+            {
+                ToggleFullscreen();
+                //cntxtMenuFullScreen_Click(null, null);
+                cntxtMenuFullScreen.Checked = true;
+            }
+            else if (Properties.Settings.Default.fullScreen == false)

```

Comment:
can `Properties.Settings.Default.fullScreen` be a non-boolean value?

Reply:
Maybe @ABranscom can comment on this, but yeah, I agree: This "else if" can simply be an "else", since we already know at this point that fullScreen is false.